llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -67,6 +67,9 @@ class RISCVInstructionSelector : public InstructionSelector {
   bool selectSelect(MachineInstr &MI, MachineIRBuilder &MIB,
                     MachineRegisterInfo &MRI) const;
 
+  bool emitFPCompare(CmpInst::Predicate Pred, Register DstReg, Register LHS,
```

Comment:
Should this function be renamed to `selectFPCompare`? It looks like we are using it to select a G_FCMP.